tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -37,18 +37,15 @@ def is_shape_type(value):
         return True
     return False
 
-def single_input_to_input_cut_info(input: [str, tuple, list, PartialShape, Type, type]):
+def single_input_to_input_cut_info(input: [tuple, list, PartialShape, Type, type]):
     """
     Parses parameters of single input to InputCutInfo.
     :param input: input cut parameters of single input
     :return: InputCutInfo
     """
     if isinstance(input, str):
-        # Parse params from string
-        node_name, shape = parse_input_value(input)
         # pylint: disable=no-member
-        return _InputCutInfo(node_name,
-                             PartialShape(shape) if shape is not None else None)
+        return _InputCutInfo(input, None)
```

Comment:
Returned back `str` to signature. I removed them in the first place because implemented solution without strings, but it turned out string args are still necessary but i forgot to return them back to signature. Thanks for noticing